lanl/docker-compose/ochami-services.yml

```diff
@@ -122,16 +123,21 @@ services:
       - BSS_DBPORT=5432
       - BSS_DBUSER=bss-user
       - BSS_DBPASS=${BSS_POSTGRES_PASSWORD} # Set in .env file
+      - BSS_AUTH_REQUIRED=true
```

Comment:
Did we remove this variable in favor of just using `BSS_JWKS_URL` to indicate that  auth was required?

Reply:
Yep. Setting `BSS_JWKS_URL` or passing `--jwks-url` will enable auth.

Reply:
Removed in  cc5ae5b.